test/unittest/QosSerializer/CMakeLists.txt

```
@@ -21,16 +21,23 @@ if(GTEST_FOUND)
     add_executable(qos_serializer_tests QosSerializerTests.cpp
         ${PROJECT_SOURCE_DIR}/src/cpp/subscriber/QosSerializer.cpp)
 
+    if(MSVC)
+        target_compile_definitions(qos_serializer_tests
+            PRIVATE _CRT_DECLARE_NONSTDC_NAMES=0 FASTDDS_STATISTICS_BACKEND_SOURCE)
+    endif(MSVC)
+
     target_include_directories(qos_serializer_tests PRIVATE
         ${GTEST_INCLUDE_DIRS} ${GMOCK_INCLUDE_DIRS}
         ${PROJECT_SOURCE_DIR}/include
-        ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}
+        ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME_UNDERSCORED}
         ${PROJECT_BINARY_DIR}/include
-        ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME}
+        ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME_UNDERSCORED}
         ${PROJECT_SOURCE_DIR}/src/cpp)
 
     target_link_libraries(qos_serializer_tests PUBLIC
-        ${GTEST_LIBRARIES} fastrtps)
+        ${GTEST_LIBRARIES} fastrtps fastcdr)
```

I don't think fastcdr is necessary
```suggestion
        ${GTEST_LIBRARIES} fastrtps)
```